docs/catalogue/cat_admin.md

```diff
@@ -13,35 +14,36 @@ Since 27 sep 2021 we have started to give catalogue schema a seperate version nu
+| 2.1 => 2.2  | ColumnType ontology and ontology_array now automatically instantiate tables of ontology type with tablename that is under refTable. These tables are not otherwise explicitly defined in the datamodel anymore. Download whole database, replace molgenis.csv and reupload in new Schema.                                                                                                                                                |
+| 2.2 => 2.3  | Alter division of AllVariables, Variables and RepeatedVariables (for Sources and Targets). Download whole database, replace molgenis.csv and reupload in new Schema.                                                                                                                                                                                                                                                                     |
+| 2.3 => 2.4  | Ontologies are placed in schema 'CatalogueOntologies' next to the catalogue itself. molgenis.csv is updated with an extra column 'refSchema' and 'columnType' ontology and ontology_array are refering to schema 'CatalogueOntologies'. It is important that 'refSchema' refers to CatalogueOntologies and this schema needs to exist before uploading molgenis.csv (or alter the ontology schema on the server and within molgenis.csv) |
+| 2.4 => 2.5  | CollectionEvents.coreVariables, Subcohorts.comorbidity, Subcohorts.counts. Add table SubcohortCounts. Download whole database, replace molgenis.csv and reupload in new Schema.                                                                                                                                                                                                                                                          | |
+| 2.5 => 2.6  | Edits to table SubcohortCounts: delete SubcohortCounts.year, SubcohortCounts.ageband, SubcohortCounts.gender, SubcohortCounts.N. Add SubcohortCounts.ageGroup, SubcohortCounts.N total, SubcohortCounts.N female, SubcohortCounts.N male. Download whole database, replace molgenis.csv and reupload in new Schema.                                                                                                                      | |
+| 2.6 => 2.7  | Add Network.type and CatalogueOntologies.NetworkTypes. To update: upload molgenis.csv and add 'h2020' and 'harmonization' to NetworkTypes                                                                                                                                                                                                                                                                                                | |
```

Comment:
seams very specific to a certain project , now all instances need to update, note sure if this is going to scale

Reply:
I don't agree it is specific. But importantly, I think we should have a way to manage changes as a 'farm' instead of 'pets' without thinking about it, unrespective if it is 'specific' or 'generic'